<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #191919;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="pc_tree">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
    background-color: #292929;
    color: white;
    padding-left: 4px;
    border: 1px solid #191919;
}

QTreeView {
	margin: 10px;
}

QTreeView::item {
	padding: 10px;
color: white;
}

QTreeView::item:hover {
    background: #003451;
}

QTreeView::item:selected:active{
    background: #003451;
}

 QScrollBar:vertical {
    background: white;
     width: 15px;
 }
 QScrollBar::handle:vertical {
    background: #0C632A;
 }

QScrollBar:horizontal {
    background: white;
    height: 15px;
}

QScrollBar::handle:horizontal {
    background: #0C632A;
}</string>
      </property>
      <property name="uniformRowHeights">
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>#</string>
       </property>
       <property name="font">
        <font/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Version</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QProgressBar" name="pbar">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
	min-height: 12px;
	max-height: 12px;
	border-radius: 6px;
	margin: 10px;
}

QProgressBar::chunk {
	border-radius: 6px;
	background-color: #0C632A;
}</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <widget class="QPushButton" name="skip">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: #a48912;
        border: none;
        color: #fff;
		padding: 7px;
		border: 2px solid #a48912;
    	border-radius: 10px;
    }

QPushButton:disabled {
        border: none;
 		border: 2px solid #0C632A;
    	border-radius: 10px;
}

QPushButton:hover {
	background-color: #957d10;
}

QToolTip { 
	color: #fff; background-color: #000; border: none; 

}

</string>
         </property>
         <property name="text">
          <string>Skip</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="start">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: #0C632A;
        border: none;
        color: #fff;
 		border: 2px solid #0C632A;
    	border-radius: 10px;
    }

 QPushButton:hover {
	background-color: #084D20;
}

QToolTip { 
	color: #fff; background-color: #000; border: none; 

}</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="cancel_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: #831721;
        border: none;
        color: #fff;
 		border: 2px solid #831721;
    	border-radius: 10px;
    }

 QPushButton:hover {
	background-color: #75141d;
}

QToolTip { 
	color: #fff; background-color: #000; border: none; 

}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="header">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
margin-bottom: 15px;</string>
         </property>
         <property name="text">
          <string>Downloading... </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="top_header">
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
